src/app/services/storage.service.ts

```diff
@@ -0,0 +1,25 @@
+import { Inject, Injectable, InjectionToken } from '@angular/core';
+
+export const BROWSER_STORAGE = new InjectionToken<Storage>('Browser Storage', {
```

Comment:
I followed the [Angular docs example](https://angular.dev/guide/di/di-in-action) here but we don't actually need this injection token at the moment. It might be useful for injecting mocks for testing in the future though so I left it in for now. (The tests for the estimator form mock the `StorageService` class.)